src/current/v23.1/security-reference/authentication.md

```diff
@@ -23,31 +23,33 @@ CockroachDB's authentication behavior is configured using a domain-specific lang
 
 A specific CockroachDB cluster's authentication behavior is configured by setting its `server.host_based_authentication.configuration` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}), using the [`SET CLUSTER SETTING` statement]({% link {{ page.version.version }}/set-cluster-setting.md %}), which accepts a single text field that must be a correctly formatted HBA manifest. Inspect the current setting with [`SHOW CLUSTER SETTING`.]({% link {{ page.version.version }}/show-cluster-setting.md %})
 
-## Currently supported authentication methods
+## Supported authentication methods
 
-Authentication Method | CockroachDB Cloud | Supported in CockroachDB Core | CockroachDB Enterprise Support
--------------|------------|-----|----
-password              |      ✓              |           ✓                    |    ✓
-[SCRAM-SHA-256]({% link {{ page.version.version }}/security-reference/scram-authentication.md %})         |      ✓              |           ✓                    |    ✓
-certificate              |      &nbsp;         |           ✓                    |    ✓
-username/password combination              |      ✓              |           ✓                    |    ✓
-[certificate]({% link {{ page.version.version }}/security-reference/transport-layer-security.md %})              |      &nbsp;         |           ✓                    |    ✓
-GSS                   |      &nbsp;         |           &nbsp;               |    ✓
+Authentication Method          | CockroachDB {{ site.data.products.cloud }} | CockroachDB {{ site.data.products.core }} | CockroachDB {{ site.data.products.enterprise }}
+-------------------------------|--------------------------------------------|-------------------------------------------|------------------------------------------------
+password                       | ✓ | ✓ | ✓
+username/password combination  | ✓ | ✓ | ✓
+[SCRAM-SHA-256][SCRAM-SHA-256] | ✓ | ✓ | ✓
+[certificate][certificate]     | ✓ | ✓ | ✓
+GSS                            |   |   | ✓
 
 All options also support the following no-op 'authentication methods', which do not perform authentication:
 
 - `reject`: unconditionally rejects the connection attempt.
 - `trust`: unconditionally accepts the connection attempt.
 
-### HBA configuration syntax
+[SCRAM-SHA-256]: {% link {{ page.version.version }}/security-reference/scram-authentication.md %}
+[certificate]: {% link {{ page.version.version }}/security-reference/transport-layer-security.md %}
 
-Each line of an Authentication Configuration (HBA) manifest defines a rule. Lines commented with `#` are ignored.
+## HBA configuration syntax
 
-For example, the following silly but easy-to-understand configuration has three rules:
+Each line of an Host-based Authentication (HBA) configuration manifest defines a rule. Lines commented with `#` are ignored.
```

Comment:
```suggestion
Each line of a Host-based Authentication (HBA) configuration manifest defines a rule. Lines commented with `#` are ignored.
```

Also FWIW, while every org seems to capitalize/hyphenate/otherwise "Host-based Authentication" differently, I think yours is the best way. 👍